pkg/spireentry/reconciler_test.go

```diff
@@ -75,3 +76,29 @@ func TestMakeEntryKey(t *testing.T) {
 		require.Equal(t, makeEntryKey(a), makeEntryKey(b))
 	})
 }
+
+func TestIsNamespaceIgnored(t *testing.T) {
+	type namespaceIgnoredTest struct {
+		ignoredNamespaces []*regexp.Regexp
+		namespace         string
+		expected          bool
+	}
+
+	regexp1, _ := regexp.Compile("s([a-z]+)re")
+	regexp2, _ := regexp.Compile("default")
+	regexps := []*regexp.Regexp{regexp1, regexp2}
```

Comment:
Rather than compile these here. let's put the uncompiled string in the test case, then compile in the for loop.